src/pages/en/guides/migrate-to-astro/from-sveltekit.mdx

```diff
@@ -26,11 +26,15 @@ SvelteKit and Astro share some similarities that will help you migrate your proj
 
 When you rebuild your SvelteKit site in Astro, you will notice some important differences:
 
-- [MPA vs SPA](/en/concepts/mpa-vs-spa/): SvelteKit is a Svelte-based SPA (single-page application). Astro sites are multi-page apps built using [`.astro` components](/en/core-concepts/astro-components/), but can also support [React, Preact, Vue.js, Svelte, SolidJS, AlpineJS, Lit](/en/core-concepts/framework-components/) and raw HTML templating.
+- [MPA vs alternatives](/en/concepts/mpa-vs-spa/): Astro sites are multi-page apps, whereas SvelteKit defaults to SPAs (single-page applications) with server-side rendering, but can also create MPAs or traditional SPAs. SvelteKit supports different modes for different pages.
+
+- [Components](/en/core-concepts/astro-components/): SvelteKit uses [Svelte](https://svelte.dev). Astro pages are built using [`.astro` components](/en/core-concepts/astro-components/), but can also support [React, Preact, Vue.js, Svelte, SolidJS, AlpineJS, Lit](/en/core-concepts/framework-components/) and raw HTML templating.
 
 - [Content-focus](/en/concepts/why-astro/): Astro was designed to excel at making content-focused websites. An existing SvelteKit app might be built for high client-side interactivity and may include items that are difficult to replicate in Astro, such as dashboards.
 
-- [Markdown-ready](/en/guides/markdown-content/): Astro includes built-in Markdown support, and includes a [special frontmatter YAML `layout` property](/en/core-concepts/layouts/#markdownmdx-layouts) used per-file for page templating. If you are converting a Sveltekit Markdown-based blog, you will not have to install a separate Markdown integration and you will not set a layout via a configuration file. You can bring your existing Markdown files, but you may need to reorganize as Astro's file-based routing does not require a folder for each page route.
+- [Markdown-ready](/en/guides/markdown-content/): Astro includes built-in Markdown support, and includes a [special frontmatter YAML `layout` property](/en/core-concepts/layouts/#markdownmdx-layouts) used per-file for page templating. If you are converting a SvelteKit Markdown-based blog, you will not have to install a separate Markdown integration and you will not set a layout via a configuration file. You can bring your existing Markdown files, but you may need to reorganize as Astro's file-based routing does not require a folder for each page route.
+
+- SvelteKit has features like [form actions](https://kit.svelte.dev/docs/form-actions) that do not have a direct equivalent in Astro. 
```

Comment:
Do you think it would make sense to fold this into the “Content-focus” bullet above that highlights how a SvelteKit app might be doing stuff that’s less easy in Astro?

Something along these lines:

```md
- [Content-focus](/en/concepts/why-astro/): Astro was designed to excel at making content-focused websites. An existing SvelteKit app might be built for high client-side interactivity and may include pages such as dashboards or use features like SvelteKit’s [form actions](https://kit.svelte.dev/docs/form-actions) that are more difficult to replicate in Astro.
```

Reply:
Yeah, that makes sense!